akka-http-core/src/test/scala/akka/http/engine/parsing/HttpHeaderParserSpec.scala

```diff
@@ -153,7 +153,8 @@ class HttpHeaderParserSpec extends WordSpec with Matchers with BeforeAndAfterAll
           |   |   ┌─o-r-i-g-i-n-:- (origin)
           |   |   |                   ┌─e-n-t-i-c-a-t-e-:- (proxy-authenticate)
           |   | ┌─p-r-o-x-y---a-u-t-h-o-r-i-z-a-t-i-o-n-:- (proxy-authorization)
-          |   | | └─r-a-n-g-e-:- (range)
+          |   | | |   ┌─a-n-g-e-:- (range)
+          |   | | └─r-e-f-e-r-e-r-:- (referer)
```

Comment:
I wonder if we shouldn't either 1) test only invariants of the tree (as far as we can identify important ones) or 2) disable these tests. 

I would for now suggest to disable these tests as AFAIK they never caught an error so far but were a maintenance burden.